# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xplanet/gcc43.patch
# Copyright (C) 2008 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- xplanet-1.2.0/src/parseColor.cpp.vanilla	2008-12-26 23:06:33.000000000 +0100
+++ xplanet-1.2.0/src/parseColor.cpp	2008-12-26 23:07:21.000000000 +0100
@@ -1,8 +1,9 @@
 #include <cstdio>
+#include <cstdlib>
 #include <fstream>
 #include <map>
 #include <sstream>
-#include <string>
+#include <cstring>
 using namespace std;
 
 #include "findFile.h"
--- xplanet-1.2.0/src/libprojection/getProjection.cpp.vanilla	2008-12-26 23:03:57.000000000 +0100
+++ xplanet-1.2.0/src/libprojection/getProjection.cpp	2008-12-26 23:04:16.000000000 +0100
@@ -1,4 +1,5 @@
 #include <cctype>
+#include <cstdlib>
 #include <cstring>
 #include <sstream>
 #include <vector>
--- xplanet-1.2.0/src/Map.cpp.vanilla	2008-12-26 23:04:43.000000000 +0100
+++ xplanet-1.2.0/src/Map.cpp	2008-12-26 23:04:55.000000000 +0100
@@ -1,4 +1,5 @@
 #include <map>
+#include <cstdlib>
 #include <sstream>
 #include <string>
 using namespace std;
--- xplanet-1.2.0/src/libannotate/Icon.cpp.vanilla	2008-12-26 22:56:15.000000000 +0100
+++ xplanet-1.2.0/src/libannotate/Icon.cpp	2008-12-26 22:56:35.000000000 +0100
@@ -1,4 +1,5 @@
 #include <sstream>
+#include <cstring>
 using namespace std;
 
 #include "findFile.h"
--- xplanet-1.2.0/src/libannotate/addArcs.cpp.vanilla	2008-12-26 22:55:27.000000000 +0100
+++ xplanet-1.2.0/src/libannotate/addArcs.cpp	2008-12-26 22:55:37.000000000 +0100
@@ -3,7 +3,7 @@
 #include <fstream>
 #include <sstream>
 #include <map>
-#include <string>
+#include <cstring>
 #include <vector>
 using namespace std;
 
--- xplanet-1.2.0/src/buildPlanetMap.cpp.vanilla	2008-12-26 23:05:35.000000000 +0100
+++ xplanet-1.2.0/src/buildPlanetMap.cpp	2008-12-26 23:05:44.000000000 +0100
@@ -1,4 +1,5 @@
 #include <cmath>
+#include <cstdlib>
 #include <cstdio>
 #include <map>
 using namespace std;
--- xplanet-1.2.0/src/libplanet/Planet.cpp.vanilla	2008-12-26 22:57:42.000000000 +0100
+++ xplanet-1.2.0/src/libplanet/Planet.cpp	2008-12-26 22:57:51.000000000 +0100
@@ -2,7 +2,7 @@
 #include <cstdio>
 #include <cctype>
 #include <sstream>
-#include <string>
+#include <cstring>
 using namespace std;
 
 #include "findFile.h"
--- xplanet-1.2.0/src/PlanetProperties.h.vanilla	2008-12-26 22:51:42.000000000 +0100
+++ xplanet-1.2.0/src/PlanetProperties.h	2008-12-26 22:55:19.000000000 +0100
@@ -1,7 +1,7 @@
 #ifndef PLANETPROPERTIES_H
 #define PLANETPROPERTIES_H
 
-#include <string>
+#include <cstring>
 #include <vector>
 
 #include "body.h"
--- xplanet-1.2.0/src/xplanet.cpp.vanilla	2008-12-26 23:08:40.000000000 +0100
+++ xplanet-1.2.0/src/xplanet.cpp	2008-12-26 23:08:48.000000000 +0100
@@ -1,5 +1,6 @@
 #include <clocale>
 #include <cstdio>
+#include <cstdlib>
 #include <iostream>
 #include <map>
 #include <sstream>
--- xplanet-1.2.0/src/xpUtil.cpp.vanilla	2008-12-26 23:07:53.000000000 +0100
+++ xplanet-1.2.0/src/xpUtil.cpp	2008-12-26 23:08:21.000000000 +0100
@@ -2,10 +2,11 @@
 #include <clocale>
 #include <cmath>
 #include <cstdio>
+#include <cstdlib>
 #include <ctime>
 #include <iostream>
 #include <sstream>
-#include <string>
+#include <cstring>
 using namespace std;
 
 #include <unistd.h>
